### DISCUSSION TITLE
Duplicated QSO

### DISCUSSION THREAD

**Discussion by ea5wa** [magicbug/Cloudlog#1071] [Show and tell]
I sometimes found duplicated QSOs (normally the last one of the day).

What could be the reason?

![image](https://user-images.githubusercontent.com/54686689/122612766-01d61580-d084-11eb-9b85-abd737f82881.png)

73 de EA5WA Juan Carlos

**Comment by dg9vh** [comment]
Maybe reload of qso logging window after logging the qso?

ea5wa Juan Carlos ***@***.***> schrieb am Fr., 18. Juni 2021,
22:25:

**Comment by ea5wa** [comment]
I don't know.... Maybe.... I have to do more tests

**Comment by magicbug** [comment]
This is likely due to refreshing the page after a POST of a QSO I try my best to handle it but only so much one can do to catch it!
